api-spec/STAC-extensions.yaml

```diff
@@ -855,6 +1008,25 @@ components:
             $ref: '#/components/schemas/item'
         links:
           $ref: '#/components/schemas/itemCollectionLinks'
+        'search:metadata':
+          type: object
+          required:
+            - next
+            - returned
+          properties:
+            next:
+              type: string
+              nullable: true
+            limit:
+              type: integer
+              nullable: true
+              minimum: 0
+            matched:
```

Comment:
I think in our discussion we decided that providing a count on search might be too expensive for many implementations. Could we make this optional?

Reply:
@scisco This hasn't been introduced here. The extension and the requirement for count was introduced in #482. With this PR, I just fixed the issue that it wasn't compiled into our full OpenAPI definition. As I don't change any semantics of the spec here, I would prefer to discuss your note in a separate issue or PR.

Reply:
Yeah, it makes sense. Thanks for pointing to the right ticket.